src/util/Constants.js

```diff
@@ -61,6 +56,7 @@ exports.DefaultOptions = {
    * WebSocket options (these are left as snake_case to match the API)
    * @typedef {Object} WebsocketOptions
    * @property {number} [large_threshold=250] Number of members in a guild to be considered large
+   * @property {Intents|number} [intents] Intents to enable for this connection
```

Comment:
This should rather be an `IntentsResolvable` for ease of use.
```suggestion
   * @property {IntentsResolvable} [intents] Intents to enable for this connection
```